packages/gatsby-source-contentful/src/__tests__/__snapshots__/gatsby-node.js.snap

```diff
@@ -2,17 +2,17 @@
 
 exports[`gatsby-node should add a new blogpost and update linkedNodes 1`] = `
 Object {
```

Comment:
Changes in this file should basically only be dropping a `c` prefix from all ids

Reply:
yes, correct :)